gradlew

```diff
@@ -1,78 +1,129 @@
-#!/usr/bin/env sh
+#!/bin/sh
```

Comment:
we use a custom gradle wrapper jar, which is loaded by the `configure.sh` script. Not sure we should update `gradlew` and `gradlew.bat` scripts ..

Reply:
Reverted